docs/api/advanced/subscribe.mdx

```diff
@@ -14,9 +14,11 @@ You can access state outside of your components and subscribe to changes.
 ```jsx
 import { subscribe } from 'valtio'
 
-// Subscribe to all state changes
+const state = proxy({ count: 0 })
+
+// Subscribe to all changes to the state proxy (and its child proxies)
 const unsubscribe = subscribe(state, () =>
-  console.log('state has changed to', state)
+  console.log('state.count has changed to', state.count)
```

Comment:
what was the confusing part before this change?

Reply:
Iirc I think I had some other edit in mind while typing that, but now forget + yeah, not necessary, so reverted.